source/_components/worldclock.markdown

```diff
@@ -34,6 +34,11 @@ name:
   required: false
   type: string
   default: Worldclock Sensor
+time_format:
+  description: The format of the time
```

Comment:
Period is missing.

Reply:
We should add a link for further details here (e.g., https://docs.python.org/3.8/library/datetime.html#strftime-strptime-behavior) as it's not obvious for a person who is not familiar with Python.

Reply:
Good point, thank you